pkg/server/test/server.go

```diff
@@ -7,7 +7,9 @@ import (
 )
 
 const (
-	DefaultResolveNodeLimit = 25
+	DefaultResolveNodeLimit        = 25
+	DefaultResolveNodeBreadthLimit = 100
+	DefaultMaxConcurrentReads      = 30
```

Comment:
Sanity check - is this an aggresively small value considering the resolve node breadth limit is more than 3x this value.

If you assume each resolution at a given level requires at least 1 database query, then setting the default max concurrent reads to a number much smaller than the breadth limit will significantly hinder concurrent evaluation.

I would recommend that we either
a) increase the max concurrent reads limit to be greater than the breadth limit
b) decrease the breadth limit to a number that is more reasonable for the aggressive read limit (with an understanding that this will noticeably impact performance).

Reply:
Do note that this list of defaults is only for tests. But we have the same numbers in `DefaultConfig()` that applies in production.

I'll go with (b). `MaxOpenConns` has been defaulted to 30 for a long time. `ResolveNodeBreadthLimit` is being introduced here, so I might as well set it to 30 now.

Reply:
Actually, hold on, `ResolveNodeBreadthLimit` didn't exist before because we were using a hardcoded value of 100 :P